libs/langchain-google-common/src/connection.ts

```diff
@@ -64,8 +64,9 @@ export abstract class GoogleConnection<
     clientLibraryVersion: string;
   }> {
     const env = await getRuntimeEnvironment();
+    const packageJson = require('../package.json');    
```

Comment:
I've discussed this with @jacoblee93 before (who can correct me if I mis-state something).
Unfortunately, while this works in Node, it doesn't work in all environments. Otherwise it would have been done in `getRuntimeEnvironment()` (which would be the best place to fix it, so it is available to all packages).

`require` itself is node-specific, but it isn't the only problem here.